docs/platforms.md

```diff
@@ -0,0 +1,20 @@
+# Tested Platforms
```

Comment:
Was this doc added accidentally?

Reply:
Yes. I noticed the mistake and added it to the right branch: https://github.com/kinvolk/headlamp/pull/256
But forgot to force push this one. Updated now.